tests/it/test_fds.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import patch
+
+from fds.services.dvc_service import DvcChoices
+from fds.utils import does_file_exist, execute_command, convert_bytes_to_string
+from tests.it.helpers import IntegrationTestCase
+
+
+class TestFds(IntegrationTestCase):
+
+    def test_init_success(self):
+        self.fds_service.init()
+        assert does_file_exist(f"{self.repo_path}/.git")
+        assert does_file_exist(f"{self.repo_path}/.dvc")
+
+    def test_status(self):
+        self.fds_service.init()
+        self.fds_service.status()
+
+    @patch("fds.services.dvc_service.DVCService._get_choice", return_value={"selection_choice": DvcChoices.ADD_TO_DVC.value})
+    def test_add(self, get_choice):
+        self.fds_service.init()
+        super().create_fake_git_data()
+        super().create_fake_dvc_data()
+        self.fds_service.add(".")
+        output = execute_command(["git", "status"], capture_output=True)
+        assert f"large_file.dvc" in convert_bytes_to_string(output.stdout)
```

Comment:
What about the git managed files?

Reply:
This check actually tests that too, because we are checking git status and making sure large_file.dvc is added to git, which wont happen if `git add .` is not called

Reply:
I can change it to make sure its a new file too

Reply:
I was actually talking about checking that the `fake_git_data` was `git add`ed